src/interface/storage.thrift

```diff
@@ -625,11 +625,11 @@ struct ScanEdgeResponse {
 struct TaskPara {
     1: common.GraphSpaceID                  space_id,
     2: optional list<common.PartitionID>    parts,
-    3: optional list<binary>                task_specfic_paras
+    3: optional list<binary>                task_specific_paras
 }
 
 struct AddAdminTaskRequest {
-    // rebuild index / flush / compact / statis
```

Comment:
This is between: `static`, `status`, and `stats`.

Reply:
It's correct.